src/crimson/os/seastore/journal/circular_bounded_journal.cc

```diff
@@ -297,10 +297,7 @@ CircularBoundedJournal::write_ertr::future<> CircularBoundedJournal::device_writ
     "overwrite in CircularBoundedJournal, offset {}, length {}",
     offset,
     length);
-  auto write_length = length < get_block_size() ? get_block_size() : length;
-  auto bptr = bufferptr(ceph::buffer::create_page_aligned(write_length));
-  auto iter = bl.cbegin();
-  iter.copy(bl.length(), bptr.c_str());
+  auto bptr = bl.begin().get_current_ptr();
```

Comment:
Look at the implementation of rebuild_aligned.  There may be multiple bufferptrs, but each is guarranteed to be aligned.  Is there a device method that can submit multiple write for each aligned buffer?

Reply:
If you really need it to be a single buffer, add a bufferptr bufferlist::get_aligned_bufferptr() const method.

Reply:
I added writev commit to address your comment.